feature/profile/build.gradle.kts

```diff
@@ -0,0 +1,18 @@
+plugins {
+    alias(libs.plugins.mifospay.android.feature)
+    alias(libs.plugins.mifospay.android.library.compose)
+}
+
+android {
+    namespace = "org.mifospay.profile"
+}
+
+dependencies {
+    implementation(projects.core.data)
+    implementation(libs.squareup.okhttp)
+    implementation(libs.compose.country.code.picker)
+    implementation(libs.compose.material)
+    implementation(libs.coil.kt.compose)
+    implementation(libs.androidx.appcompat)
+    implementation(projects.mifospay)
```

Comment:
remove this, mifospay is parent module and this profile is child module.

Reply:
we need  'implementation(projects.mifospay)'  for BuildConfig in :-   val uri = FileProvider.getUriForFile(
        Objects.requireNonNull(context),
        BuildConfig.APPLICATION_ID + ".provider", file
    )

Reply:
just do hard coded package name for now